server.py

```diff
@@ -84,9 +84,10 @@ def investigate_issues(investigate_request: InvestigateRequest):
             investigate_request.context.get("robusta_issue_id")
         )
 
-        instructions = dal.get_resource_instructions(
-            "alert", investigate_request.context.get("issue_type")
-        )
+        # resource_instructions = dal.get_resource_instructions(
```

Comment:
did you remove this on purpose, or just forgot to handle?
I think this remove fetching the alert instructions

Reply:
That is a mistake and the next line after the commented ones should be removed.